### PR TITLE
add position relative to table element

### LINKED ISSUE #416: Unexpected behavior on table horizontal scrolling
Hi @shadcn! Many thanks for this repo, amazing work 👏

Just noticed a strange behavior on the table component. When the content of the table is larger than the screen, a white space the size of the overflow is created. I have tried several things to fix this but nothing that works properly so far.

https://github.com/shadcn/ui/assets/15351304/1e2d097a-866d-449a-ac80-164dc82f796f

### PR DESCRIPTION
**Pull request by PixeledCode**
closes #416

### CODE REVIEW

File: apps/www/components/ui/table.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const Table = React.forwardRef<
   <div className="w-full overflow-auto">
```
Comment (its-monotype) [inline apps/www/components/ui/table.tsx:9]:
```suggestion
  <div className="w-full overflow-auto relative">
```

File: apps/www/components/ui/table.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ const Table = React.forwardRef<
...
     <table
       ref={ref}
-      className={cn("w-full caption-bottom text-sm", className)}
+      className={cn("w-full caption-bottom text-sm relative", className)}
```
Comment (its-monotype) [inline apps/www/components/ui/table.tsx:12]:
```suggestion
      className={cn("w-full caption-bottom text-sm", className)}
```

**Comment by PixeledCode** [comment]
@its-monotype is the PR okay now?

**Comment by PixeledCode** [comment]
@shadcn looks like the linting check failed because of tailwind classes order. I was not able to find a list on where to place the `position` class. Any suggestions?

**Comment by moreei** [comment]
My plugin autocorrected the order to this:

`<div className="relative w-full overflow-auto">`

**Comment by PixeledCode** [comment]
@shadcn I think it should be mergeable now

**Comment by kyb3r** [comment]
Thanks for the fix, I was struggling with this in my app. 

But how did you know what to do. Amazing.

**Comment by PixeledCode** [comment]
opened new issue #1370

**Pull request closed without merging**